prob/torus_cbc/build.py

```diff
@@ -408,7 +423,9 @@
 bhl.config.set_cparm('QUADRANT_SYMMETRY', QUAD)
 
 # EOS
-bhl.config.set_cparm("EOS_TYPE", EOS_TYPE)
+bhl.config.set_cparm("EOS", EOS_TYPE)
```

Comment:
So one small change I was thinking of: Here the name of the parameter is `EOS` which saves which type of EoS is going to be used. Also, around line 100, there is 
```
if RELTABLE:
    bhl.report_var('EOS','RELTABLE')
elif GAMTABLE:
    bhl.report_var('EOS','GAMTABLE')
else:
    bhl.report_var('EOS','GAMMA')
```
where the parameter name is `EOS`. I was wondering whether the latter parameter can be changed to something like `EOS_VAL` to keep the two parameter names different?

Reply:
no those should be the same. `report_var` doesn't do anything internal in the code. It prints a choice of variable to the build output.